.env.slack

```diff
@@ -0,0 +1,11 @@
+GOOGLE_APPLICATION_CREDENTIALS=./google-credentials/service-account.json
```

Comment:
are the changes to the next three files necessary? why are they part of this PR?